client/packages/core/tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "outDir": "dist",
     "rootDir": "src",
     "sourceMap": true,
+    "inlineSources": true,
```

Comment:
It _seems_ like this is okay: 

https://chatgpt.com/share/66ec6a75-9ca0-8003-ae2a-916d472ba29a